src/components/structures/UserSettings.js

```diff
@@ -359,6 +363,8 @@ module.exports = React.createClass({
     },
 
     _renderLabs: function () {
+        if (this.props.enableLabs === false) return null;
```

Comment:
But if it's not specified, it will get the value from defaultProps

On 5 August 2016 17:19:32 BST, David Baker <notifications@github.com> wrote:
>> @@ -359,6 +363,8 @@ module.exports = React.createClass({
>>      },
>>  
>>      _renderLabs: function () {
>> +        if (this.props.enableLabs === false) return null;
>
>Intention was to make it default to enabled if it wasn't specified,
>which I'ver clarified in a comment.
>
>---
>You are receiving this because you were assigned.
>Reply to this email directly or view it on GitHub:
>https://github.com/matrix-org/matrix-react-sdk/pull/393/files/f8d02c1551ff9392be45a0cfed0fcd33d5078048#r73719512